composer.json

```diff
@@ -19,7 +19,8 @@
   "require-dev": {
     "wp-coding-standards/wpcs": "1.2.1",
     "dealerdirect/phpcodesniffer-composer-installer": "0.4.4",
-    "phpcompatibility/php-compatibility": "9.1.1"
+    "phpcompatibility/php-compatibility": "9.1.1",
+    "xwp/wp-dev-lib": "dev-feature/composer-first"
```

Comment:
@westonruter This can be set to `^1.0` now.

Reply:
Thanks. I've pinned it to `1.0.0` in 744077a9 since Renovate will open PRs to bump when new versions are available.